### ISSUE TITLE
CA certificate mount seems incompatible with RHEL7

### ISSUE THREAD

**Issue by redradrat** [kubermatic/kubermatic#6407]
**User Story**

With kubermatic 2.15, the hostDir, used for mounting CA certs from the respective seed node, is set to "/etc/ssl/certs".

https://github.com/kubermatic/kubermatic/blob/54cb7fc6ccd29da817c583a922f14f7af563fe5b/pkg/resources/resources.go#L1061-L1084

Is there any additional way to configure this? Because this path does not hold true for RHEL 7 for example, that is officially supported, as per doc. While RHEL7 stores it's certificates at /etc/pki and maintains a symlink to the aforementioned hardcoded directory, hostpath-based volumemounts do not seem to support following symlinked paths.

https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/security_guide/sec-shared-system-certificates#:~:text=In%20Red%20Hat%20Enterprise%20Linux,%2Dtrust%2Dsource%2F%20directories.

In conclusion, all calls from controller manager to well-known external public services fail, as their certificates are not trusted. 

**Acceptance criteria**

* configurable cert mount dir
* node-type specific mount
* magic is always an option I guess?

**Comment by redradrat** [comment]
Adding an additional volume mount like so:

```

 		{ 
 			Name: HostPKIVolumeName, 
 			VolumeSource: corev1.VolumeSource{ 
 				HostPath: &corev1.HostPathVolumeSource{ 
 					Path: "/etc/pki", 
 					Type: &doc, 
 				}, 
 			}, 
 		} 
```
```
		{
			Name:      HostPKIVolumeName,
			MountPath: "/etc/pki",
			ReadOnly:  true,
		}
```

would fix the issue in a way compatible wit RHEL and debian bases.

**Comment by xrstf** [comment]
@redradrat Can you try the latest 2.15 release? We fixed this recently.

**Comment by redradrat** [comment]
@xrstf thx for the info. Will check. But the change looks exactly like what we did in our patch. So should be fine.

**Comment by kron4eg** [comment]
I'd assume the check went good.
